CHANGES/528.feature

```diff
@@ -0,0 +1 @@
+Added support for syncing signatures using docker API extension.
```

Comment:
now that all the pieces are implemented for the signature sync, can you add/update the docs?